airbyte-webapp/src/config/types.ts

```diff
@@ -21,12 +21,13 @@ declare global {
 export interface Config {
   segment: { token: string; enabled: boolean };
   apiUrl: string;
-  connectorBuilderApiUrl: string;
+  #connectorBuilderApiUrl: string; #FIXME: Uncomment this when enabling the connector-builder
   oauthRedirectUrl: string;
   healthCheckInterval: number;
   version?: string;
   integrationUrl: string;
   launchDarkly?: string;
+  #connectorBuilderUrl: string; #FIXME: Uncomment this when enabling the connector-builder
```

Comment:
```suggestion
  connectorBuilderApiUrl: string; #FIXME: Uncomment this when enabling the connector-builder
  oauthRedirectUrl: string;
  healthCheckInterval: number;
  version?: string;
  integrationUrl: string;
  launchDarkly?: string;
```

Reply:
Weird diff but basically I think you can just undo the changes in this file as well